rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -348,21 +350,24 @@ def run_all_post_decoding_rules(
         assert self._check_correct_types(decoded_events)
         return decoded_events
 
-    def decode_transaction(
+    def _decode_transaction(
             self,
             transaction: EvmTransaction,
             tx_receipt: EvmTxReceipt,
-    ) -> list['EvmEvent']:
+    ) -> tuple[list['EvmEvent'], bool]:
```

Comment:
you are adding a new return value to the function. What is it for? Why have you not changed the docstring?